dataprofiler/tests/profilers/test_int_column_profile.py

```diff
@@ -460,3 +460,35 @@ def test_profile_merge_no_bin_overlap(self):
                                     'Profiles have no overlapping bin methods '
                                     'and therefore cannot be added together.'):
             profiler1 + profiler2
+            
+    def test_profile_merge_with_different_options(self):
+        # Creating first profiler with default options
+        options = IntOptions()
+        options.max.is_enabled = False
+        options.min.is_enabled = False
+
+        data = [2, 4, 6, 8]
+        df = pd.Series(data).apply(str)
+        profiler1 = IntColumn("Int", options=options)
+        profiler1.update(df)
+        profiler1.match_count = 0
+
+        # Creating second profiler with separate options
+        options = IntOptions()
+        options.min.is_enabled = False
+        data2 = [10, 15]
+        df2 = pd.Series(data2).apply(str)
+        profiler2 = IntColumn("Int", options=options)
+        profiler2.update(df2)
+
+        # Asserting error when adding 2 profilers with different options
```

Comment:
assert raises error?

Reply:
also should with a `with self.assertWarnsRegex(...)`

Reply:
Fixed